td-agent/templates/package-scripts/td-agent/deb/postinst

```diff
@@ -57,10 +57,10 @@ update_conffile() {
     # changing to the right user. Then, these directories were created with
     # root permission. The following lines fix that problem.
     if [ -d "/var/log/<%= project_name %>/buffer/" ]; then
-        chown -R <%= project_name %>:<%= project_name %> /var/log/<%= project_name %>/buffer/
+        chown <%= project_name %>:<%= project_name %> /var/log/<%= project_name %>/buffer/*
```

Comment:
I'm not sure there are grand-child directories or not.
So I think just suppressing the warning is better in this case.

Reply:
Ah, I will change PR.